arxivdigest/frontend/views/admin.py

```diff
@@ -58,9 +58,9 @@ def toggleSystem(systemID, state):
         mail = {'to_address': sys['email'],
                 'subject': 'System Activation',
                 'template': 'systemActivation',
-                'data': {'name': sys['contact_name'],
+                'data': {'name': sys['firstname'] + " " + sys['lastname'],
                          'key': sys['api_key'],
-                         'link': request.url_root}}
+                         'link': 'arxivdigest.org'}}
```

Comment:
This should be a config setting, not hardcoded.
I think `web_address` in config.json could be moved one level up (from interleave_config) as that's a global setting.